plugins/prefetch/plugin.cc

```diff
@@ -460,7 +542,7 @@ contHandleFetch(const TSCont contp, TSEvent event, void *edata)
         /* first-pass */
         if (!config.isExactMatch()) {
           data->_fetchable = state->acquire(data->_cachekey);
-          PrefetchDebug("request is %s fetchable", data->_fetchable ? " " : " not ");
+          PrefetchDebug("request is %s fetchable", data->_fetchable ? "" : "not");
```

Comment:
More appropriate fix would be removing spaces around %s instead of changing the literals.

Reply:
reverted the change